src/modules/translations/services/translations.service.ts

```diff
@@ -0,0 +1,31 @@
+import { CACHE_MANAGER, Inject, Injectable, Logger } from '@nestjs/common';
+import { Cache } from 'cache-manager';
+import { get } from 'lodash';
+
+import { Translations } from '../types';
+
+import { GET_SITE_VARIABLES_BY_NAME } from './queries.gql';
+
+import { DataService } from '~modules/data/services/data.service';
+@Injectable()
+export class TranslationsService {
+	private logger: Logger = new Logger(TranslationsService.name, { timestamp: true });
+
+	constructor(
+		private dataService: DataService,
+		@Inject(CACHE_MANAGER) private cacheManager: Cache
+	) {}
+
+	public async getTranslations(): Promise<Translations> {
+		let translations: Translations = await this.cacheManager.get('translations');
+		if (!translations) {
+			const response = await this.dataService.execute(GET_SITE_VARIABLES_BY_NAME, {
+				name: `translations-frontend`,
```

Comment:
Keep the same naming convention as with the TOS:
```suggestion
				name: `TRANSLATIONS_FRONTEND`,
```

![image](https://user-images.githubusercontent.com/1710840/160372375-f27cf9d7-cddd-4223-8f9f-0d3c998f0244.png)

Reply:
updated